src/plugins/console/public/application/models/legacy_core_editor/legacy_core_editor.ts

```diff
@@ -297,6 +297,11 @@ export class LegacyCoreEditor implements CoreEditor {
     });
   }
 
+  unregisterKeyboardShortcut(command: string) {
+    // @ts-ignore
```

Comment:
why do we need to ignore this next line? 🤔

Reply:
Getting the following ts error here ` Property 'removeCommand' does not exist on type 'CommandManager'` but we have access to it.